apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -255,6 +257,20 @@ const sendMessagesForExpiredMeterVerificationDate = async ({ context, metersWith
     }))
 }
 
+const sendSubmitMeterReadingsPushNotificationsTaskFn = async (context = null) => {
```

Comment:
it's too many wrappers. Could you please move all this logic inside the apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotificationsTask.js

Reply:
fixed